src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/ReadsFilter.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+
+/**
+ * A simple filter for reads. Pulled into its own class due to Java limitations.
+ */
+public final class ReadsFilter extends DoFn<Read, Read>   {
```

Comment:
Could we make this a `PTransform<PCollection<Read>, PCollection<Read>>` that wraps a `DoFn<Read, Read>`? This would be a bit better from the perspective of the pipeline skeleton -- but if it's a non-trivial change, then we can address it in a future branch.